code/datums/wounds/cuts.dm

```diff
@@ -0,0 +1,287 @@
+
+/datum/wound/brute/cut/remove_wound(ignore_limb, replaced)
+	if(!replaced && highest_scar)
+		already_scarred = TRUE
+		highest_scar.lazy_attach(limb)
+	return ..()
+
+/datum/wound/brute/cut/get_examine_description(mob/user)
+	if(!current_bandage)
+		return ..()
+
+	var/bandage_condition = ""
+	// how much life we have left in these bandages
+	switch(current_bandage.absorption_capacity)
+		if(0 to 1.25)
+			bandage_condition = "nearly ruined "
+		if(1.25 to 2.75)
+			bandage_condition = "badly worn "
+		if(2.75 to 4)
+			bandage_condition = "slightly bloodied "
+		if(4 to INFINITY)
+			bandage_condition = "clean "
+	return "<B>The cuts on [victim.p_their()] [limb.name] are wrapped with [bandage_condition] [current_bandage.name]!</B>"
+
+/datum/wound/brute/cut/receive_damage(wounding_type, wounding_dmg, wound_bonus)
+	if(wounding_type == WOUND_SHARP)
+		blood_flow += 0.05 * wounding_dmg
+
+/datum/wound/brute/cut/handle_process()
+	if(victim.reagents && victim.reagents.has_reagent(/datum/reagent/toxin/heparin))
+		blood_flow += 0.5 // old herapin used to just add +2 bleed stacks per tick, this adds 0.5 bleed flow to all open cuts which is probably even more strong as long as you can cut them first
+	else if(victim.reagents && victim.reagents.has_reagent(/datum/reagent/medicine/coagulant))
+		blood_flow -= 0.25
+
+	if(current_bandage)
+		if(clot_rate > 0)
+			blood_flow -= clot_rate
+		blood_flow -= current_bandage.absorption_rate
+		current_bandage.absorption_capacity -= current_bandage.absorption_rate
+		if(current_bandage.absorption_capacity < 0)
+			victim.visible_message("<span class='danger'>Blood soaks through \the [current_bandage] on [victim]'s [limb.name].</span>", "<span class='warning'>Blood soaks through \the [current_bandage] on your [limb.name].</span>", vision_distance=COMBAT_MESSAGE_RANGE)
+			QDEL_NULL(current_bandage)
+			treat_priority = TRUE
+	else
+		blood_flow -= clot_rate
+
+	if(blood_flow > highest_flow)
+		highest_flow = blood_flow
+
+	if(blood_flow < minimum_flow)
+		if(demotes_to)
+			replace_wound(demotes_to)
+		else
+			to_chat(victim, "<span class='green'>The cut on your [limb.name] has stopped bleeding!</span>")
+			qdel(src)
+
+/* BEWARE, THE BELOW NONSENSE IS MADNESS. bones.dm looks more like what I have in mind and is sufficiently clean, don't pay attention to this messiness */
+
+/datum/wound/brute/cut/treat(obj/item/I, mob/user)
+	if(istype(I, /obj/item/gun/energy/laser))
+		las_cauterize(I, user)
+	else if(I.tool_behaviour == TOOL_CAUTERY || I.get_temperature() > 300)
+		tool_cauterize(I, user)
+	else if(istype(I, /obj/item/stack/medical/gauze))
+		bandage(I, user)
+	else if(istype(I, /obj/item/stack/medical/suture))
+		suture(I, user)
+
+/datum/wound/brute/cut/try_handling(mob/living/carbon/human/user)
+	if(user.pulling != victim || user.zone_selected != limb.body_zone || user.a_intent == INTENT_GRAB)
+		return FALSE
+
+	if(!isfelinid(user))
+		return FALSE
+
+	lick_wounds(user)
```

Comment:
okay, i laughed but i don't think making felinids objectively better at medical care via having more options to heal wounds without equipment is a good idea at all

giving species objective bonuses to job related stuff like medical is extremely iffy design wise

Reply:
alrgith after looking at the numbers here
it removes 0.5 blood loss from a wound upon usage, with what appears to be no cooldown
I could probably spam cure any cut with this

Reply:
like, for reference, the biggest cut starts you at 4.25 blood loss if I'm reading the code right, meaning I can smash down a cut with 5-10 licks, which due to how our setups work for click code I can probably do in about 5 seconds with a bunch of stacked up clicks